modules/apps/frontend-js/frontend-js-a11y-web/src/main/resources/META-INF/resources/components/Violation.tsx

```diff
@@ -90,25 +85,26 @@ function Violation({
 					</PanelSection>
 					<PanelSection title={Liferay.Language.get('occurrences')}>
 						<ClayList className="list-group-flush">
-							{nodes.map((_occurrence, index) => {
-								const occurrenceName = `${Liferay.Language.get(
-									'occurrence'
-								)} ${String(index + 1)}`;
+							{nodes.map((target, index) => {
+								const name = Liferay.Util.sub(
```

Comment:
Is this duplicate of https://github.com/liferay-frontend/liferay-portal/pull/1295/files#r679993929?

Reply:
Why would it be duplicated? these are different things, this sub will generate the name based on the rendering, and in this link we will generate the name based on the Overlay interaction, these are different places, the name is passed to the `Occurrence` component like you had done before, so I had to do the same on the other side so that I could render the occurrence that the Overlay clicked.